unittest/test_700c_eq.sh

```diff
@@ -4,9 +4,9 @@
 
 results=$(mktemp)
 
-c2enc 700C ../raw/kristoff.raw /dev/null --var 2> $results
+../build_linux/src/c2enc 700C ../raw/kristoff.raw /dev/null --var 2> $results
 var=$(cat $results | sed -n "s/.*var: \([0-9..]*\) .*/\1/p")
```

Comment:
That will break on systems that don't used `build_linux` or a different path to `c2sim`.  The path to `c2sim` should be set up in the ctest section of `CMakeLists.txt`:
```
     add_test(NAME test_700c_eq
             COMMAND sh -c "cd ${CMAKE_CURRENT_SOURCE_DIR}/unittest;
                            PATH=$PATH:${CMAKE_CURRENT_BINARY_DIR}/src ./test_700c_eq.sh")
```

This test has worked fine to date.  Can you pls describe why we need this change, or why it isn't working for you?